### PR TITLE
Added a module in Section 5 for Importing Content at the end of Secti…

### PR DESCRIPTION
**Pull request by bangelic**
…on 5 called Repository sets

Cherry-pick into:

* [ ] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

File: guides/common/modules/con_repository-sets.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+endif::[]
+
+ifdef::orcharhino[]
+Content from upstream as well as from Canonical, Oracle, Red Hat, SUSE, and custom content in {Project} have similarities:
+endif::[]
+
+Repository sets show available repositories to each host. You can enable and disable the repositories.
+If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host. However, you also have the option to show all.
+
+The default states for repository sets are Enabled and Disabled. You can override the default state by using the action menus on each table row by changing Override to disabled, Override to enabled, and Reset to default. The Status Flag will indicate that the repository has been overridden.
```
Comment (melcorr) [inline guides/common/modules/con_repository-sets.adoc:19]: How can a repository have two default states that are in contradiction?

File: guides/common/modules/con_repository-sets.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Both Red Hat content and {customcontent} in {Project} have similarities:
+endif::[]
+
+ifdef::foreman-el,katello[]
+Both Red Hat content and non-RH {customcontent} in {Project} have similarities:
+endif::[]
+
+ifdef::orcharhino[]
+Content from upstream as well as from Canonical, Oracle, Red Hat, SUSE, and custom content in {Project} have similarities:
+endif::[]
```
Comment (melcorr) [inline guides/common/modules/con_repository-sets.adoc:14]: I think these sections are copied over in error.

**Review by melcorr — CHANGES_REQUESTED** [review]
Hey Brian, 

Thanks for this! 
You're on the way. 
Perhaps this would be best written as a procedure? You have some introductory content, but then the description of labels causes the text to diverge in style from the rest of the guide. 
Please also remember to use the advice here: https://cbea.ms/git-commit/
We also follow IBM Style when it comes to referring to Graphical User Interface Elements. This means that we follow the highlighting recommendations there. If you look at other sections of the guide, you'll see that buttons and menu items etc have certain highlighting. 
This is a good first attempt, but there's a bit more work needed!

File: guides/common/modules/proc_repository-sets.adoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[id="Repository_Sets_{context}"]
+= Repository Sets in {Project}
+
+Repository sets show available repositories to each host. If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_repository-sets.adoc:4]:
```suggestion
Repository sets show available repositories to each host.
If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.
```

File: guides/common/modules/proc_repository-sets.adoc [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* to choose from a list of available hosts.
+. Select a host and navigate to the *Repository sets* tab.
+. On the *Repository sets* tab, there is a set of available repositories available to each host with a default status of *Enabled* and *Disabled*.
+. You can override the default status by using the action menus on each table row by changing the status to *Override to disabled*, *Override to enabled*, or *Reset to default*.
+. Navigate to the *Overview* tab to *Content view details* to view the environment for the host.
+[NOTE]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_repository-sets.adoc:12]:
```suggestion
+
[NOTE]
```

**Review by maximiliankolb — COMMENTED** [review]
Thanks for your PR! I think the procedure lacks a verb in the title, anchor, and file name.

I also made two small suggestions & I think @melcorr will take care of everything language-wise.

File: guides/common/modules/proc_repository-sets.adoc (hunk 3 above)
Comment (melcorr) [reply guides/common/modules/proc_repository-sets.adoc:4]: @bangelic what's happening here is that we have a rule about writing one sentence per line.

**Review by melcorr — CHANGES_REQUESTED** [review]
Big improvment @bangelic 
You're getting closer and closer. 
A few more comments. 
Please, please, please, learn to write good commit messages. Yours are way too long https://chris.beams.io/posts/git-commit/ - I'll keep pasting this. If you don't get it, let me know. 
Take a look at Chapter 7 of the IBM Style Guide. It has everything you need for describing GUI Interfaces.

**Pull request closed without merging**